scripts/memcache-stats.sh

```diff
@@ -1,8 +1,12 @@
 #!/bin/bash
 argument="$1"
+
+IP_ADDR=$(hostname -I | grep -oP "\b10(\.\d+){3}" )
+CLEAN_IP_ADDR=$( echo "$IP_ADDR" | tr '.' '-' )
```

Comment:
The spacing inside `$()` is inconsistent on these two lines. My preference would be no spaces.

Reply:
https://github.com/iFixit/pipe-to-graphite/pull/16/commits/fb416fb0a17b63beeb76641c679e2cf4609edec9